beetsplug/chroma.py

```diff
@@ -135,7 +133,8 @@ def __init__(self):
         })
 
         if self.config['auto']:
-            self.register_listener('import_task_start', fingerprint_task)
+            self.register_listener('import_task_start',
+                                   partial(fingerprint_task, self._log))
```

Comment:
Unfortunately that does not work: `beets/plugins.py:422` uses `inspect.getargspec` which does not recognize `partial` as a function... I'm not sure when that hack is needed though.